src/transformers/tokenization_utils_base.py

```diff
@@ -3275,10 +3275,15 @@ def prepare_seq2seq_batch(
             Otherwise, input_ids, attention_mask will be the only keys.
         """
         warnings.warn(
-            "`prepare_seq2seq_batch` is deprecated and will be removed in version 5 of 🤗 Transformers. Use the "
-            "regular `__call__` method to prepare your inputs and the tokenizer under the `as_target_tokenizer` "
-            "context manager to prepare your targets. See the documentation of your specific tokenizer for more "
-            "details",
+            """
+`prepare_seq2seq_batch` is deprecated and will be removed in version 5 of HuggingFace Transformers. To continue using
+it copy-n-paste the internals of this function. The gist of which is:
+
+model_inputs = tokenizer(src_texts, ...) with tokenizer.as_target_tokenizer(): labels = tokenizer(tgt_texts, ...)
+model_inputs["labels"] = labels["input_ids"]
```

Comment:
This is not properly formatted so won't be readable once outputted as warning. Might need to be put in a separate string with a preceding `# docstyle-ignore` comment to disable the formatting script.

Reply:
why do you suggest that? I tested it and the output is:

```
/mnt/nvme1/code/huggingface/transformers-prepare_seq2seq_batch/src/transformers/tokenization_utils_base.py:3277: FutureWarning: 
`prepare_seq2seq_batch` is deprecated and will be removed in version 5 of 🤗 Transformers. Instead copy-n-paste the internals of this function. The gist of which is:

model_inputs = tokenizer(src_texts, ...)
with tokenizer.as_target_tokenizer():
    labels = tokenizer(tgt_texts, ...)
model_inputs["labels"] = labels["input_ids"]

See the documentation of your specific tokenizer for more details on the specific arguments to the tokenizer of choice

  warnings.warn("""
Test with normal tokenizer: 10
```

Reply:
oh, I see, the formatter removed the new lines... arghhh... let me see